code/obj/submachine/cooking.dm

```diff
@@ -927,11 +928,14 @@ table#cooktime a#start {
 					F.reagents.add_reagent("sugar", 20)
 					qdel( P )
 				if (/obj/item/reagent_containers/food/drinks/milk)
-					new/obj/item/reagent_containers/food/snacks/condiment/cream(src.loc)
-					qdel( P )
+					if (P.reagents.has_reagent("milk") && P.reagents.get_reagent_amount("milk") >= MIN_FLUID_INGREDIENT_LEVEL)
```

Comment:
you don't need to test if it has the reagent

Reply:
Gotcha, have removed it